worker/subprojects/openssl.wrap

```diff
@@ -1,14 +1,14 @@
 [wrap-file]
-directory = openssl-3.0.2
-source_url = https://www.openssl.org/source/openssl-3.0.2.tar.gz
-source_filename = openssl-3.0.2.tar.gz
-source_hash = 98e91ccead4d4756ae3c9cde5e09191a8e586d9f4d50838e7ec09d6411dfdb63
-patch_filename = openssl_3.0.2-1_patch.zip
-patch_url = https://wrapdb.mesonbuild.com/v2/openssl_3.0.2-1/get_patch
-patch_hash = 762ab4ea94d02178d6a1d3eb63409c2c4d61315d358391cdac62df15211174d4
+directory = openssl-3.0.7
+source_url = https://www.openssl.org/source/openssl-3.0.7.tar.gz
+source_filename = openssl-3.0.72.tar.gz
+source_hash = 83049d042a260e696f62406ac5c08bf706fd84383f945cf21bd61e9ed95c396e
+patch_filename = openssl_3.0.7-1_patch.zip
```

Comment:
Just in case, shouldn't path version match opensssl version?

Reply:
Not sure what you mean exactly, can you elaborate?

Reply:
source_filename has version 3.0.72.
path_filename has version 3.0.7-1. I would expect 3.0.72-1 but it maybe perfectly ok, just in case.

Reply:
Patch name is only derived from version specified in wrapdb and has no direct connection to source code, source code can come from anywhere.

Also see https://github.com/mesonbuild/wrapdb/pull/806/files/308f3fc80531983f049087bf7efde4f1a223ba5e#diff-b0cd4975c76ae72ddd8217c684fedf270ef7c0b43df7a923367c913024c828e2

Reply:
The source_filename has a typo, "72" instead of "7" -- as you can see, it doesn't match the remote download url.

Fortunately, the only actual effect of that is that the cache filename will be weird. It's still downloaded from the right place, checksummed with the right checksum, extracted to the right directory, and the wrapped subproject builds correctly.

The cache filename doesn't much matter IMHO.